src/Mutator/Util/Mutator.php

```diff
@@ -30,8 +31,12 @@ public function __construct(MutatorConfig $config)
 
     final public function shouldMutate(Node $node): bool
     {
-        if (!$this->mutatesNode($node)) {
-            return false;
+        try {
```

Comment:
I'm a bit confused: why is it necessary here?

Reply:
This is meant to catch errors like the one in #382. Infection *should* never error during that step, so if it does we want to let the user know about it, and give the details that user can hand to us.